test/topology_experimental_raft/test_tablets.py

```diff
@@ -109,3 +109,40 @@ async def test_scans(manager: ManagerClient):
         assert r.c == r.pk
 
     await cql.run_async("DROP KEYSPACE test;")
+
+
+@pytest.mark.asyncio
+async def test_bootstrap(manager: ManagerClient):
+    logger.info("Bootstrapping cluster")
+    servers = [await manager.server_add(), await manager.server_add(), await manager.server_add()]
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', "
+                  "'replication_factor': 1, 'initial_tablets': 32};")
+    await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
+
+    logger.info("Populating table")
+
+    keys = range(256)
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO test.test (pk, c) VALUES ({k}, {k});") for k in keys])
+
+    async def check():
+        logger.info("Checking table")
+        rows = await cql.run_async("SELECT * FROM test.test;")
+        assert len(rows) == len(keys)
+        for r in rows:
+            assert r.c == r.pk
+
+    logger.info("Adding new server")
+    await manager.server_add()
+
+    await check()
+
+    logger.info("Adding new server")
+    await manager.server_add()
+
+    await check()
+    time.sleep(5) # Give load balancer some time to do work
```

Comment:
Let's loop here waiting for the migration to complete, peeking at the tablets table. We can keep check()ing meanwhile

Reply:
It's not that simple. The fact that tablets are not in migration doesn't mean the load balancing is finished.